cmake/testing.cmake

```diff
@@ -142,6 +142,10 @@ f3d_test(NAME TestVolumeComp DATA vase_4comp.vti ARGS -vb --comp=3 LONG_TIMEOUT)
 f3d_test(NAME TestVolumeDirect DATA vase_4comp.vti ARGS -vb --comp=-2 LONG_TIMEOUT)
 f3d_test(NAME TestVolumeCells DATA waveletArrays.vti ARGS -vb --cells LONG_TIMEOUT)
 f3d_test(NAME TestVolumeNonScalars DATA waveletArrays.vti ARGS -vb --scalars=RandomPointScalars LONG_TIMEOUT)
+f3d_test(NAME TestTextureNormal DATA WaterBottle.glb ARGS --geometry-only --texture-normal=${CMAKE_SOURCE_DIR}/testing/data/normal.png --normal-scale=0.1 DEFAULT_LIGHTS)
+f3d_test(NAME TestTextureMaterial DATA WaterBottle.glb ARGS --geometry-only --texture-material=${CMAKE_SOURCE_DIR}/testing/data/red.jpg --roughness=1 --metallic=1 DEFAULT_LIGHTS) # roughness and metallic have no effect ?
+f3d_test(NAME TestTextureColor DATA WaterBottle.glb ARGS --geometry-only --texture-base-color=${CMAKE_SOURCE_DIR}/testing/data/albedo_mod.png --color=1,1,0 --opacity=0.1 --depth-peeling DEFAULT_LIGHTS) #may be impacted by https://github.com/f3d-app/f3d/issues/332
```

Comment:
@Meakk PTAL

Reply:
Ok, it seems like the `opacity` argument isn't taken into account when using a texture.

Reply:
It does have an effect though. What should be the right behavior here in your opinion ?

Reply:
The effect is coming from the texture. The correct behavior should be multiplication, just like metallic and roughness.

Reply:
no, with ou without opacity is not the same result

Reply:
you are right, it is not multiplying

Reply:
added a note, outside of the scope of this MR.